src/main/webapp/view/view.js

```diff
@@ -14,12 +14,17 @@
 
 const PARAM_ID = 'id';
 const PARAM_VIDEO_ID = 'video-id';
+const ENDPOINT_LECTURE = '/lecture';
 
 /* exported LECTURE_ID */
 window.LECTURE_ID = getLectureId();
 window.VIDEO_ID = getVideoId();
 
-initialize();
+// TODO: Ask if calling .then() on async is bad practice.
```

Comment:
TODOs should be actionable code changes, this seems like something that needs to be done before the review.

Reply:
go/js-practices/async-functions top of page three does something similar